provider/storage_benchmark.py

```diff
@@ -335,7 +335,10 @@ def run(self, cmd_options='-a', timeout=1800):
 
 class FioLinuxCfg(object):
     def __init__(self, params, session):
-        fio_pkg = params.get("fio_pkg", 'fio-3.13-48-ga819.tar.bz2')
+        if params.get('os_variant') == 'rhel9':
```

Comment:
Should we get guest version then install latest fio on rhel9 guest?

Reply:
If there is a network issue, for instance, to download fio timeout at that time, it will lead to the test error, so I suggest we could prepare it in the tp-qemu repo so that we could guarantee the case run smoothly.

Reply:
Checked  os_variant, it represents guest os version,